### DISCUSSION TITLE
Incorrect list of projects in “Used By”

### DISCUSSION THREAD

**Discussion by z3dev** [community/community#14638] [Other Feature Feedback, Questions, & Ideas]
The JSCAD project is built and maintained as a set of sub-modules using Lerna. As such, the list of projects that use JSCAD is just non-existent and really sad.

https://github.com/jscad/OpenJSCAD.org

GitHub needs to support these type of projects better.

**Comment by Noggog** [comment]
I'm in a similar boat, but my problem exists even within just my own repository.

It would be nice if the `Used By` dropdown allowed for more than one package to be selected.   Perhaps more of a checklist dropdown?

I have a library setup where I have a structure like this:
```
         Core Libs
     /               \
 SubLibA          SubLibB
      \              /
   One-Size-Fits-All-lib
```
Ideally users `SubLibA` or `SubLibB`, but if they're unsure or just new to things, the `One-Size-Fits-All` is there to make it easy (at the expense of importing more than they need).

However, I can only list one of these libraries to be counted in my `Used By` counter.   I'm currently hooking it to `One-Size-Fits-All`, and have a nice counter number.  However, I'm starting to encourage users to use `SubLibA` or `SubLibB` more, at which point my counter will actually start to decrease, haha.
